src/constants.js

```diff
@@ -11,3 +11,4 @@ export const USER_INTERFACE_ID = 'user-interface';
 export const START_QUIZ_BUTTON_ID = 'start-quiz-button';
 export const ANSWERS_LIST_ID = 'answers-list';
 export const NEXT_QUESTION_BUTTON_ID = 'next-question-button';
+export const COUNTDOWN_TIME_ID = 'time';
```

Comment:
Since values of these constants are just kind of a replica of the variable name, shall we also replace this value with `countdown-time`?
```suggestion
export const COUNTDOWN_TIME_ID = 'countdown-time';
```